sass/variables/_sizing.scss

```diff
@@ -65,6 +65,7 @@ $type-indent: 2em;
 
 // UI components
 $medium-icon-font-size: 25px;
+$large-icon-font-size: $medium-icon-font-size * 1.75;
```

Comment:
Spoke with Susan and we're making the accordion angles 35px, so I've added a variable specific to that

I'm removing `$large-icon-font-size` because I think we'll have more sizes that are specific to one thing, and having specific one-off UI sizes is so simple to do that it doesn't cause a significant amount of extra work so I think it's fine (at least as long as the UI list is not too long and difficult to maintain)